src/views/ApartmentApp/components/StudentApplication/components/OffCampusSection/components/ProgramListItem/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Divider,
+  FormControl,
+  Grid,
+  Input,
+  InputLabel,
+  ListItem,
+  ListItemText,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+const ProgramListItem = ({
+  disabled,
+  index,
+  applicantProfile,
+  offCampusProgram,
+  departments,
+  onOffCampusInputChange,
+}) => {
+  const [applicantDepartmentValue, setDepartmentValue] = useState(''); // Department drop-down menu value
+
+  useEffect(() => {
+    setDepartmentValue(offCampusProgram ?? '');
+  }, [offCampusProgram]);
+
+  const handleDepartmentInputChange = (event) => {
+    if (event.target.value !== null) {
+      let newApplicantDepartmentValue = event.target.value;
+      onOffCampusInputChange(newApplicantDepartmentValue, index);
+    }
+  };
+
+  const departmentOptions = departments.map((departmentName) => (
+    <MenuItem value={departmentName} key={departmentName}>
+      {departmentName}
+    </MenuItem>
+  ));
+
+  return (
+    <React.Fragment>
+      <ListItem key={applicantProfile.AD_Username} className={'list-item'}>
+        <Grid container alignItems="flex-end" spacing={1}>
+          <Grid item xs={12} sm={4}>
+            <ListItemText primary={applicantProfile.fullName} className={'list-item'} />
+          </Grid>
+          <Grid item xs={12} sm={8}>
+            <FormControl fullWidth>
+              <InputLabel>Department</InputLabel>
+              <Select
+                disabled={disabled}
+                value={applicantDepartmentValue}
```

Comment:
Here is where you use the prop instead of the state variable
```suggestion
                value={offCampusProgram}
```